gems/aws-sdk-core/lib/aws-sdk-core/xml/parser/engines/ox.rb

```diff
@@ -16,7 +16,7 @@ def parse(xml)
           Ox.sax_parse(
             @stack, StringIO.new(xml),
             :convert_special => true,
-            :skip => :skip_white
+            :skip => :skip_return
```

Comment:
Here is an example code:

```ruby
require 'ox'

class Sample < ::Ox::Sax
  def text(value); p value; end
end

xml = <<~XML
<?xml version="1.0" encoding="UTF-8"?><list>
  <value>
 with linebreak
</value>
  <value>with  consecutive spaces  </name>
</list>

XML

handler = Sample.new

puts '== skip_white =='
Ox.sax_parse(handler, StringIO.new(xml), { skip: :skip_white })
puts '== skip_return =='
Ox.sax_parse(handler, StringIO.new(xml), { skip: :skip_return })
puts '== skip_off =='
Ox.sax_parse(handler, StringIO.new(xml), { skip: :skip_off })
puts '== skip_none =='
Ox.sax_parse(handler, StringIO.new(xml), { skip: :skip_none })
```

The output like below:

```
== skip_white ==
" with linebreak "
"with consecutive spaces "
== skip_return ==
"\n with linebreak\n"
"with  consecutive spaces  "
== skip_off ==
"\n  "
"\n with linebreak\n"
"\n  "
"with  consecutive spaces  "
"\n"
== skip_none ==
"\n  "
"\n with linebreak\n"
"\n  "
"with  consecutive spaces  "
```

As you can see above, `:skip_return` works as expected.